cypress/e2e/cautionaryAlertCreate.cy.js

```diff
@@ -0,0 +1,76 @@
+import { seedDatabase } from '../helpers/DbHelpers';
+import CautionaryAlertsPageObjects from '../pageObjects/cautionaryAlertsPage';
+import PersonPageObjects from '../pageObjects/personPage';
+
+const personPage = new PersonPageObjects();
+const caPage = new CautionaryAlertsPageObjects();
+
+const tags = ["@cautionary-alerts", "@authentication", "@common", "@root"];
+
+describe('Create Cautionary Alerts', { tags: tags }, () => {
+    beforeEach(() => {
+        cy.login();
+        seedDatabase();
+
+        cy.intercept('POST', `${Cypress.env('CAUTIONARY_ALERT_ENDPOINT')}/cautionary-alerts/`, (req) => {
+            req.on('after:response', (res) => {
+                if (res.statusCode === 200) {
+                    const alertId = res.body.alertId;
+                    // TODO: Figure out how to delete cautionary alert
+                    // queueDeleteCautionaryAlertWithId(alertId);
+                }
+            });
+        }).as('addCautionaryAlert');
+    });
+
+    it('can create a cautionary alert for a person', () => {
+        cy.getPersonFixture().then((person) => {
+            // Navigate to person page and click on Add cautionary alert link
+            personPage.visit(person.id);
+            caPage.addCautionaryAlertLink().click();
```

Comment:
I know this isn't in scope for this PR but addCautionaryAlertLink should really be attached to the Person & Tenure page objects. No action needed in this PR, just complaining